src/Compilers/Core/Portable/Compilation/IExpression.cs

```diff
@@ -828,6 +829,80 @@ public static BinaryOperandsKind GetBinaryOperandsKind(BinaryOperationKind kind)
         }
     }
 
+    public interface IConversion
```

Comment:
1. Can you explain how these extension methods would work?  I can't see how, given the definition of IConversionExpression, how an extension method could answer the questions that i'm exposing through properties.
2. The value of hte interface is that it provides a *common* way to talk about conversions, regardless of if they come through IOperation or if they come through SemanticModel/Compilation.  

The latter is very important to the IDE.  We don't want to handle conversions differently.   It would be like having IOperation return a new entity that represented Symbols.  That would be very awkward and unpleasant.  We would far prefer to avoid that.